website/docs/r/storage_bucket.markdown

```diff
@@ -101,6 +125,22 @@ The `CORS` object supports the following:
 
 * `max_age_seconds` (Optional) Specifies time in seconds that browser can cache the response for a preflight request.
 
+* `server_side_encryption_configuration` (Optional) A configuration of server-side encryption configuration (documented below)
+
+The `server_side_encryption_configuration` object supports the following:
+
+* `rule` (Required) A single object for server-side encryption by default configuration. (documented below)
```

Comment:
~~I can't tell what's the meaning of "by default configuration" here, please explain~~
Maybe change to "Rules for encryption. Only a single rule for encryption is supported for now"?

Reply:
Although `rule` is nested there could only be one rule (in AWS as well).